stack.yaml

```diff
@@ -1,6 +1,6 @@
 resolver: lts-12.26
 
-ghc-options: {"$locals": -ddump-to-file -ddump-hi}
+# ghc-options: {"$locals": -ddump-to-file -ddump-hi}
```

Comment:
Is this supposed to be part of this PR? I have no idea what it does, just asking :-)

Reply:
gets rid of .dump-hi files everywhere, just for stackers